xtext-website/_posts/releasenotes/2020-03-03-version-2-21-0.md

```diff
@@ -8,6 +8,10 @@ published: false
 
 Xtext 2.21.0 is mainly a maintenance release.
```

Comment:
Xtext 2.21 is a maintenance release.

Reply:
well.. did not change that => should be updated separately